modules/custom/openy_search/openy_google_search/openy_google_search.install

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * @file
+ * Module installation file.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function openy_google_search_install() {
+
+  if (\Drupal::service('path.alias_storage')->aliasExists('/search', 'en')) {
+    return;
+  }
+ $installed = \Drupal::service('module_installer')->install(['openy_prgf_google_search']);
+  if (!$installed) {
+    return;
+  }
+
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $node_storage = \Drupal::entityTypeManager()->getStorage('node');
+
+  $search_paragraph = $paragraph_storage->create([
+    'type' => 'google_search',
+    'field_prgf_block' => [
+      0 => [
+        'plugin_id' => 'google_search_block',
+        'plugin_configuration' => [
+          'id' => 'google_search_block',
+          'label' => 'Google search',
+          'provider' => 'openy_google_search',
+          'label_display' => 'hidden',
+        ],
+      ],
+    ],
+  ]);
+  $search_paragraph->save();
+
+  $page = $node_storage->create([
+    'type' => 'landing_page',
+    'title' => 'Search results',
+    'field_lp_layout' => 'one_column_clean',
+    'langcode' => 'en',
+    'uid' => '1',
+    'field_content' => [
+      [
+        'target_id' => $search_paragraph->id(),
+        'target_revision_id' => $search_paragraph->getRevisionId(),
+      ],
+    ],
+    'status' => 1,
+  ]);
+  $page->save();
+  \Drupal::service('path.alias_storage')->save('/node/' . $page->id(), '/search', 'en');
```

Comment:
I think we can define an alias in the node data array and pass to the `create()` method. Something like:
```
'path' => [
        'pathauto' => PathautoState::SKIP,
        'alias' => $alias,
      ],
```